blockchain/chainstore.go

```diff
@@ -71,10 +74,23 @@ func NewChainStore() (IChainStore, error) {
 	return store, nil
 }
 
+func (c *ChainStore) GetChainID() uint64 {
+	var id uint64
+	idByte, err := c.Get([]byte{byte(SYS_ID)})
+	if err == leveldb.ErrNotFound {
```

Comment:
use idByte == nil instead of err == leveldb.ErrNotFound, to ensure an no-empty id will be retured